logstash-core-plugin-api/logstash-core-plugin-api.gemspec

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "logstash-core-plugin-api/version"
+# This is completely valid, when we do a `gem build` the gemspec is 
+# converted to a text file without any ruby code.
+require_relative "../logstash-core/lib/logstash-core/version"
+
+Gem::Specification.new do |gem|
+  gem.authors       = ["Elasticsearch"]
```

Comment:
I think we all missed the rebranding :P 
https://github.com/ph/logstash/blob/b140886a71d72b8947771dd896d84c0a53a43117/logstash-core-event-java/logstash-core-event-java.gemspec#L7